jkube-kit/generator/java-exec/src/test/java/org/eclipse/jkube/generator/javaexec/JavaExecGeneratorMainClassDeterminationTest.java

```diff
@@ -164,18 +165,15 @@ public void testMainClassDeterminationFromFatJar(
 
         JavaExecGenerator generator = new JavaExecGenerator(generatorContext);
 
-        final List<ImageConfiguration> images = new ArrayList<>();
-
-        List<ImageConfiguration> customized = generator.customize(images, false);
-
-        assertEquals("1 images returned", 1, customized.size());
-
-        ImageConfiguration imageConfig = customized.get(0);
+        List<ImageConfiguration> customized = generator.customize(new ArrayList<>(), false);
 
-        assertEquals("Image name", "TheFatJarImageName", imageConfig.getName());
-        assertNull("Main Class is NOT set as environment variable#",
-                imageConfig.getBuildConfiguration().getEnv().get(JavaExecGenerator.JAVA_MAIN_CLASS_ENV_VAR));
+        assertThat(customized).singleElement()
+                .hasFieldOrPropertyWithValue("name", "TheFatJarImageName")
+                .extracting(ImageConfiguration::getBuildConfiguration)
+                .extracting(BuildConfiguration::getEnv)
+                .asInstanceOf(InstanceOfAssertFactories.MAP)
+                .as("Main Class is NOT set as environment variable#")
+                .doesNotContainEntry(JavaExecGenerator.JAVA_MAIN_CLASS_ENV_VAR, null);
```

Comment:
Could we use literals here instead of referencing the `JavaExecGenerator` constants?